config/prod.exs

```diff
@@ -3,4 +3,6 @@ use Mix.Config
 config :bors, :server, BorsNG.GitHub.Server
 config :bors, :oauth2, BorsNG.GitHub.OAuth2
 
+config :bors, :local_merge?, {:system, :boolean, "USE_LOCAL_MERGE", false}
```

Comment:
Which one of these environments do we use for the portal-suite repo?

Reply:
prod; dev and test don't hit GitHub at all